### PR TITLE
Add AV1, OPUS, Vorbis, E-AC3, MP3 support

### PR DESCRIPTION
**Pull request by jonathanhperry**
Add:
- AV1
- Opus
- Vorbis
- E-AC3
- MP3

### PR Checklist 

- [x] Code is PEP8 conform (https://www.python.org/dev/peps/pep-0008/)
- [x] PR contains tests for code changes

### CODE REVIEW

**Comment by ioreper** [comment]
@dominic-miglar Can you please take a look?

File: bitmovin/resources/models/codecconfigurations/av1_codec_configuration.py [hunk 1]
```diff
@@ -0,0 +1,275 @@
...
+        else:
+            raise InvalidTypeError('colorConfig has to be of type ColorConfig')
+
+    @property
+    def keyPlacementMode(self):
+        return self._keyPlacementMode
+
+    @keyPlacementMode.setter
+    def keyPlacementMode(self, new_key_placement_mode):
+        if new_key_placement_mode is None:
```
Comment (moser4035) [inline bitmovin/resources/models/codecconfigurations/av1_codec_configuration.py]: shouldn't `_keyPlacementMode` be set to None here?

File: bitmovin/resources/models/codecconfigurations/av1_codec_configuration.py [hunk 2]
```diff
@@ -0,0 +1,275 @@
...
+            raise InvalidTypeError('Invalid type {} for rateControlMode: must be either str or AV1RateControlMode.'
+                                   .format(type(new_rate_control_mode)))
+
+    @property
+    def adaptiveQuantMode(self):
+        return self._adaptiveQuantMode
+
+    @adaptiveQuantMode.setter
+    def adaptiveQuantMode(self, new_adaptive_quant_mode):
+        if new_adaptive_quant_mode is None:
```
Comment (moser4035) [inline bitmovin/resources/models/codecconfigurations/av1_codec_configuration.py]: Shouldn't `self._adaptiveQuantMode` set to `None` here?

Comment (jonathanhperry) [reply bitmovin/resources/models/codecconfigurations/av1_codec_configuration.py]: I see mixed patterns across the codec configurations, some just return and others set the private property. Makes sense to set the private property in the settor, so will update.

**Pull request merged**